plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/TestGlobalTransactionMySqlConnectorTest.java

```diff
@@ -17,18 +17,19 @@
 
 import static io.trino.plugin.mysql.MySqlQueryRunner.createMySqlQueryRunner;
 import static io.trino.tpch.TpchTable.CUSTOMER;
+import static io.trino.tpch.TpchTable.LINE_ITEM;
 import static io.trino.tpch.TpchTable.NATION;
 import static io.trino.tpch.TpchTable.ORDERS;
 import static io.trino.tpch.TpchTable.REGION;
 
-public class TestGlobalTransactionMySqlIntegrationSmokeTest
-        extends BaseMySqlIntegrationSmokeTest
+public class TestGlobalTransactionMySqlConnectorTest
```

Comment:
Do we need to run all tests here? It would be nice to have a smoke test (not `AbstractIntegrationSmokeTest`) suite that we could use here. Can you please create an issue to create such suite and add TODO here?

Reply:
https://github.com/trinodb/trino/issues/7019

Reply:
Thanks! please link to this in the code here & and in `TestJdbcCachingQueries`